json-endpoints/class.wpcom-json-api-list-posts-v1-1-endpoint.php

```diff
@@ -382,7 +382,9 @@ function callback( $path = '', $blog_id = 0 ) {
 						if ( ! isset( $return[$key] ) ) {
 							$return[$key] = (object) array();
 						}
-						$return[$key]->next_page = $this->build_page_handle( $last_post, $query );
+							if ( isset( $last_post['ID'] ) ) {
+								$return[ $key ]->next_page = $this->build_page_handle( $last_post, $query );
+							}
```

Comment:
Indenting if off I think, no?

```suggestion
						if ( isset( $last_post['ID'] ) ) {
							$return[ $key ]->next_page = $this->build_page_handle( $last_post, $query );
						}
```